content/deploy/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Deploy
+slug: /deploy
+---
+
+# Deploy
+
+Get all the information you need to deploy your app and share it with your users.
+
+<InlineCalloutContainer>
+    <InlineCallout
+        color="lightBlue-70"
+        icon="book"
+        bold="Concepts."
+        href="/deploy/concepts"
+    >Understand the basics of app deployment.</InlineCallout>
+    <InlineCallout
+        color="lightBlue-70"
+        icon="cloud"
+        bold="Streamlit Community Cloud."
+        href="/deploy/streamlit-community-cloud"
+    >Deploy your app on our free platform and join a community of developers who share their apps around the world.</InlineCallout>
+    <InlineCallout
+        color="lightBlue-70"
+        icon="ac_unit"
+        bold="Streamlit in Snowflake"
+        href="https://docs.snowflake.com/developer-guide/streamlit/about-streamlit"
+    >Deploy your app in Snowflake to enjoy a role-based access controlled environment with all your data in one unified, global system.</InlineCallout>
```

Comment:
Added to next commit. I said "...for a secure, enterprise-grade environment" since it's both a development and deployment environment but we're kind of emphasizing the deployment part here.